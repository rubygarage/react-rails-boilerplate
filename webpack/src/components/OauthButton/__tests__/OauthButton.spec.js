import React from 'react';
import { shallow } from 'enzyme';
import OathButton from 'components/OauthButton';

describe('<OathButton />', () => {
  const defaultProps = {
    color: 'blue-dark',
    onClick: jest.fn(),
    type: 'icon-fb',
  };

  it('renders oauth button', () => {
    const component = shallow(<OathButton {...defaultProps} />);

    expect(component).toMatchSnapshot();
  });
});
