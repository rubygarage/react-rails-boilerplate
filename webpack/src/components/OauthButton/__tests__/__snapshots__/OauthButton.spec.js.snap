// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OathButton /> renders oauth button 1`] = `
<Button
  className="mb-20"
  color="blue-dark"
  href=""
  onClick={[Function]}
  size="md"
  to=""
>
  <Icon
    className="social-icon"
    type="icon-fb"
  />
</Button>
`;
